.github/workflows/docker.yml

```diff
@@ -55,13 +75,54 @@ jobs:
           echo "${{ secrets.DOCKER_BOT_TOKEN }}" | docker login -u dukeroboticsbot --password-stdin
           docker buildx imagetools create --tag dukerobotics/robosub-ros:latest dukerobotics/robosub-ros:amd64 dukerobotics/robosub-ros:arm64
 
+  latest:
+    runs-on: ubuntu-latest
+    needs: [arm64, amd64]
+    env:
+      IMAGE_NAME: dukerobotics/robosub-ros:latest
+      FILE_NAME: latest.tar.gz
+      ARTIFACT_NAME: latest
+    steps:
+      - name: Get arm64 image
+        uses: actions/download-artifact@v1
+        with: 
+          name: arm64
+      - name: Load arm64 image
+        run: |
+          docker load < ./arm64/arm64.tar.gz
+          rm -rf arm64
+      - name: Get amd64 image
+        uses: actions/download-artifact@v1
+        with: 
+          name: amd64
+      - name: Load amd64 image
+        run: |
+          docker load < ./amd64/amd64.tar.gz
+          rm -rf amd64
+      - name: Save image
+        run: |
+          docker manifest create ${IMAGE_NAME} dukerobotics/robosub-ros:arm64 dukerobotics/robosub-ros:amd64
```

Comment:
can't use manifest here, use 
```
docker buildx imagetools create --tag dukerobotics/robosub-ros:latest dukerobotics/robosub-ros:amd64 dukerobotics/robosub-ros:arm64
```

Reply:
never mind, I'm not sure whether that will correctly use the local images and it may also just push the new manifest to docker hub. I think we'd be best off just using amd64 in the simulation Dockerfile since a lot of these manifest tools haven't been fully developed yet.